config/locales/idv/en.yml

```diff
@@ -5,6 +5,10 @@ en:
       masked_ssn: secure text, starting with %{first_number} and ending with %{last_number}
     buttons:
       cancel: Cancel and return to your profile
+      change_address_label: Change address
+      change_label: Update
```

Comment:
Were we intending to change the text here from "Change" to "Update" ?

The links on the right:

![image](https://user-images.githubusercontent.com/1779930/177328093-4ea90b34-c8b8-4a3d-8d6b-4fe1a087d0cc.png)

**Edit:** I see this update is proposed in the Figma attached to LG-6285. Two follow-up points:

- Do we also need to revise the accessible `aria-label`s to match? Since my understanding is that those are supposed to be essentially the same, but more contextualized for assistive technology ([link purpose](https://www.w3.org/WAI/WCAG21/Understanding/link-purpose-in-context.html))
- Similar to other comment, I expect these would need to be updated in the partner-facing Figma designs

Reply:
I'll also aim to get the Spanish and French translations for all of these label changes (the visible and aria labels) included as we do IPP content translations

Reply:
Updated the English aria labels in 87f7c0c75cbb6fcad78575cef776aa0d31685766. I've updated LG-6337 to track getting translations for the Spanish and French translations of this change (eg Change => Update in Spanish and French for the various labels and buttons)